src/radiation/moments.cpp

```diff
@@ -882,20 +988,18 @@ TaskStatus MomentCalculateOpacities(T *rc) {
           /// opacity so that we can
```

Comment:
Yes, I think I just forgot to remove that TODO when I added those grid variables. 

My original thinking was that the integrated opacities required for the moment source update could be filled by a task that uses the MOCMC samples (i.e. `MOCMCCalculateOpacities`) or by a simple task like this one (`MomentCalculateOpacities`) that just assumes some gray opacity based on the properties of the moments (or by doing something dumb and assuming all neutrinos are 10 MeV).

Reply:
OK that makes sense, thanks.

I agree it would be nice to just have some task that updates opacities, either via some mean opacity or MOCMC samples. For at least some methods for MOCMC fluid sourcing, though, (in particular if we iterate on the fluid temperature and so need updated opacities each iteration) I had trouble reconciling opacities-via-separate-task with updating opacities each iteration. Admittedly we're not doing that right now, but we might want to later. One could still always call the opacities task prior to any interactions and then update opacities further inside the interaction loop if necessary... Maybe something to be refined later.